### PR TITLE
Remote Command Execution Using UDP Client Server Model (& Merging Branch Locally)

### PR DESCRIPTION
**Pull request by NVombat**
#### Description of Change
Added Remote Command Execution Using UDP Client Server Model & Merging Branch to be up to date for further contributions

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C/pull/930"></a>

### CODE REVIEW

**Comment by NVombat** [comment]
I Added Remote Command Execution Using UDP Client Server Model

File: .vscode/settings.json [hunk 1]
```diff
@@ -4,4 +4,7 @@
     "editor.formatOnPaste": true,
     "editor.formatOnType": true,
     "files.insertFinalNewline": true,
+    "files.associations": {
+        "stdlib.h": "c"
+    },
```
Comment (Panquesito7) [inline .vscode/settings.json:9]: Please revert the changes made here. :)

**Review by Panquesito7 — COMMENTED** [review]
Great work on this! 🚀 Would you like to add self-test cases so we can verify your program works properly (**see the [typical structure of a program](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) as a reference**)? Let us know if you need any help here or in our [**Discord server**](https://discord.gg/c7MnfGFGa6). 😃

**Comment by NVombat** [comment]
I have verified the working of the programs with multiple use cases so it doesnt require testcases in my opinion. I compiled the client and server using "gcc -o server/client server.c/client.c" and then ran the programs "./server ./client"... It works without any errors

**Comment by NVombat** [comment]
And another reason I cant add test cases is because it is a client server program and can only be tested by compiling and running - With User Input... Thus It is best left as is

**Review by Panquesito7 — APPROVED** [review]
Amazing work! 👀🚀 Thanks so much for your contribution! 😄👍🎉

After this PR is merged, you can ask for the
contributor role in our [**Discord server**](https://discord.gg/c7MnfGFGa6)! 🚀
